lib/sentry/client.ex

```diff
@@ -289,7 +289,7 @@ defmodule Sentry.Client do
     |> Map.from_struct()
     |> update_if_present(:mechanism, &Map.from_struct/1)
     |> update_if_present(:stacktrace, fn %Interfaces.Stacktrace{frames: frames} ->
-      %{frames: Enum.map(frames, &Map.from_struct/1)}
+      %{frames: frames && Enum.map(frames, &Map.from_struct/1)}
```

Comment:
Should `frames` be present at all if it's null, according to Sentry docs?

Reply:
@savhappy need to figure this ☝️ out before merging 🙃

Reply:
Yep yep did some digging in the docs and it says the `frames` attribute is required if stacktrace is present so we should remove the property entirely if `frames: nil`? @whatyouhide thoughts?

Reply:
Nice, perfect!